packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatAdapter.ts

```diff
@@ -338,6 +340,20 @@ export interface CallWithChatAdapterManagement {
     content: string,
     options?: SendMessageOptions | /* @conditional-compile-remove(file-sharing-acs) */ MessageOptions
   ): Promise<void>;
+  /* @conditional-compile-remove(rich-text-editor-image-upload) */
+  /**
+   * Upload an inline image for a message.
+   *
+   * @beta
+   */
+  uploadImage(image: ArrayBuffer | Blob, imageFilename: string): Promise<UploadChatImageResult>;
```

Comment:
it might be a better idea to add our wrapper on top of `UploadChatImageResult` from `@azure/communication-chat` and use it here as part of return promise instead of exposing chat sdk type as part of our public api